index.html

```diff
@@ -48,7 +50,200 @@ <h1 class="h1">I’m Patrick Ndimukulaga Glad to see you!</h1>
+            <div class="port">
+              <h4>Tonic</h4>
+              <ul class="ul">
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <div class="q-text">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </div>
+          <div class="mastery">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn btn-mastery" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="title grid">
+        <div>
+          <img src="./images/Snapshoot_Portfolio3.png" alt="portfolio"/>
+        </div>
+        <div class="title-text">
+          <div class="t-text">
+            <div class="port">
+              <h4>Multi-Post Stories</h4>
+
+              <ul class="ul">
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <div class="q-text">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </div>
+          <div class="mastery">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+
+          <button class="btn btn-mastery" type="button">See Project</button>
+        </div>
+      </div>
+    </div>
+
+    <section class="about">
+      <div class="container">
+        <div>
+          <h2 class="about_header">About<br />Myself</h2>
+          <p class="win">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+        </div>
+
+        <section>
+          <h2 class="media_header">LET'S CONNECT</h2>
+          <ul class="media_ul">
+            <li class="media_li"><a href="#" target="_blank"><img src="./images/Github-icon.svg" alt="Github icon"/></a>
+            </li>
+            <li class="media_li"><a href="#" target="_blank"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
+            </li>
+            <li class="media_li"><a href="#" target="_blank"><img src="./images/Vector.svg" alt="Vector icon"/></a>
+            </li>
+            <li class="media_li"><a href="#" target="_blank"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
+            </li>
+            <li class="media_li"><a href="#" target="_blank"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
+            </li>
+          </ul>
+        </section>
```

Comment:
- [ ] Kindly remove the section tag wrapping the let's connect  and social media icons, this will make your website more semantic and easily readable by browsers